GW2EIEvtcParser/EncounterLogic/Raids/W7/PeerlessQadim.cs

```diff
@@ -312,11 +364,10 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
 
         internal override void ComputePlayerCombatReplayActors(AbstractPlayer p, ParsedEvtcLog log, CombatReplay replay)
         {
-            // fixated
+            // Fixated
             var fixated = p.GetBuffStatus(log, FixatedQadimThePeerless, log.FightData.FightStart, log.FightData.FightEnd).Where(x => x.Value > 0).ToList();
             foreach (Segment seg in fixated)
             {
-                replay.Decorations.Add(new CircleDecoration(120, seg, "rgba(255, 80, 255, 0.3)", new AgentConnector(p)));
```

Comment:
Why remove this?

Reply:
Looked a bit much now that there are the orb aoes on tank and we have the fixation overhead

![6159](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/5d3bf333-5704-4dc1-832d-fd42f59b771a)

![6160](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/2e89c015-9746-4666-a41c-843284fb44df)